jest.config.js

```diff
@@ -4,7 +4,6 @@ module.exports = {
     clearMocks: true,
     restoreMocks: true,
     moduleNameMapper: {
-        'react-intl-locale-data': '<rootDir>/node_modules/react-intl/locale-data/en.js',
```

Comment:
You might have to add https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#jest to line 13. Also there is another jest.config.js inside scripts folder I think, double check if changes are needed there. That one is used for visual tests only.